lib/osv/debian_version_cache.py

```diff
@@ -0,0 +1,18 @@
+"""Caches the debian package first version dataframe"""
+
+import pandas as pd
+
+debian_version_cache = pd.read_pickle(
```

Comment:
Is there a large benefit to using pandas here to deserialize and index this? We should aim to minimize the number of third party dependencies we require. 

What if we did the heavy lifting in the tool that generates this index, which just dumps a JSON file that we download and deserialize here as a dict?

Reply:
There's not really a large benefit, so yeah can do the json option.